lib/screens/app/wallet/dashboard.dart

```diff
@@ -489,4 +492,80 @@ class _DashboardState extends State<Dashboard> {
       ),
     );
   }
+
+  Widget buildSendReceiveButton() {
+
+    final double width = MediaQuery.of(context).size.width;
+    final double textScaleFactor = width >= 320 ? 1.0 : 0.8;
+    // 2.37 comes from original screen width 375 / 158 original button width.(Provided on figma file)
+    final double widthFactor = width / 2.37;
+    // 0.304 comes form original button height 48 / 158 original button width.
+    final double heightFactor = (widthFactor * 0.304);
+    final double heightRadiusFactor = heightFactor / 1.37;
+    final double widthRadiusFactor = widthFactor / 45.14;
+
+
+    return Padding(
```

Comment:
This is very complex and unnecessary


`FlatButton(
                    textColor: Colors.white,
                    child: Text("Send"),
                    onPressed: () {},
                    shape: RoundedRectangleBorder(
                        borderRadius: BorderRadius.only(
                            topLeft: Radius.circular(2),
                            topRight: Radius.circular(30),
                            bottomLeft: Radius.circular(30),
                            bottomRight: Radius.circular(2))),
                    color: Colors.green,
                  ),`
                  
                  Will accomplish the same result.